lib/ts/recipe/emailpassword/index.ts

```diff
@@ -31,16 +32,17 @@ export default class Wrapper {
         return EmailPassword.getInstanceOrThrow().signOut();
     }
 
-    static async isEmailVerified(userContext?: any): Promise<{
+    static async isEmailVerified(input?: { userContext?: any }): Promise<{
```

Comment:
Expose all other email verification functions as well in recipes that use email verification under them

Reply:
Done